sdk/healthcareapis/azure-mgmt-healthcareapis/tests/recordings/test_mgmt_healthcareapis.test_healthcareapis.yaml

```diff
@@ -0,0 +1,798 @@
+interactions:
+- request:
+    body: '{"kind": "fhir-R4", "location": "westus2", "properties": {"accessPolicies":
+      [{"objectId": "c487e7d1-3210-41a3-8ccc-e9372b78da47"}, {"objectId": "5b307da8-43d4-492b-8b66-b0294ade872f"}],
+      "cosmosDbConfiguration": {"offerThroughput": 1000}, "authenticationConfiguration":
+      {"authority": "https://login.microsoftonline.com/abfde7b2-df0f-47e6-aabf-2462b07508dc",
+      "audience": "https://azurehealthcareapis.com", "smartProxyEnabled": true}, "corsConfiguration":
+      {"origins": ["*"], "headers": ["*"], "methods": ["DELETE", "GET", "OPTIONS",
+      "PATCH", "POST", "PUT"], "maxAge": 1440, "allowCredentials": false}}}'
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '599'
+      Content-Type:
+      - application/json; charset=utf-8
+      User-Agent:
+      - python/3.6.8 (Linux-4.9.125-linuxkit-x86_64-with-Ubuntu-18.04-bionic) msrest/0.6.8
+        msrest_azure/0.4.34 azure-mgmt-healthcareapis/0.1.0 Azure-SDK-For-Python
+      accept-language:
+      - en-US
+    method: PUT
+    uri: https://management.azure.com/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/test_mgmt_healthcareapis_test_healthcareapis965511fd/providers/Microsoft.HealthcareApis/services/myapimrndxyz?api-version=2018-08-20-preview
```

Comment:
should the real subscription id be changed to `00000000-0000-0000-0000-000000000000` ?